adobecom/cc
# MWPW-154899 - Pilot preloading marquee dependencies

* Preload known marquee dependencies if in LCP section
* Not in scope: MEP awareness

**Resolves:** [MWPW-154899](https://jira.corp.adobe.com/browse/MWPW-154899)

**Test URLs:**
- Before: https://main--cc--adobecom.hlx.live/products/illustrator?martech=off&mep=off
- After: https://marqpreload--cc--adobecom.hlx.live/products/illustrator?martech=off&mep=off

**How to test:**
Observe in the network calls that all marquee dependencies load before things that are historically in front of blocks: placeholders, etc.

**Additional context:**
This is a pilot to see if this feature should be elevated to Milo at a later date. I'm a little suspect this will add any performance gains in low bandwidth scenarios, but it's worth trying.
